R/plot_with_settings.R

```diff
@@ -365,11 +365,6 @@ plot_with_settings_srv <- function(id,
       if (plot_type() == "gg" && dblclicking) {
         plot_r() +
           ggplot2::coord_cartesian(xlim = ranges$x, ylim = ranges$y, expand = FALSE)
-      } else if (plot_type() == "grob") {
-        # calling grid.draw on plot_r() is needed;
-        # otherwise the plot will not re-render if the user triggers the zoom in or out feature of the browser.
-        grid::grid.newpage()
-        grid::grid.draw(plot_r())
```

Comment:
why it didn't worked here?

Reply:
When the` 'grob'` is re-rendered for a modal pop-up, it's not displaying the image as expected. Initially, I suspected that the changes I made in teal.code for the `'qenv'`  might be the cause, particularly due to the use of 'pdf(null.file())'. However, upon further investigation, it appears that this is not the root of the problem.

I still not sure whats the issue but when `plot_reactive()` is called second time it does not render it for grob object